### PR TITLE
Update Podfile.lock to Cocoapods 1.0.1

### PR DESCRIPTION
**Pull request by petey3**

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -14,4 +14,6 @@ SPEC CHECKSUMS:
   RxCocoa: a2baaf3462b989559a51dce59d78fbebb8fe0afa
   RxSwift: 829b2843e9eddc77a02cb3f5117f0d55f8f646f1
 
-COCOAPODS: 0.39.0
+PODFILE CHECKSUM: 74c18fd468262f8bff0bacdd6e8261b90a972c0f
+
+COCOAPODS: 1.0.0
```
Comment (Benuuu) [inline Podfile.lock]: 1.0.1 is the current latest version.  Can you install with that instead?

Comment (petey3) [inline Podfile.lock]: yep

**Comment by Benuuu** [comment]
:shipit:

**Comment by petey3** [comment]
👎  botched it, needed a second :shipit:  and remembered right after I merged.

**Comment by Benuuu** [comment]
next time.

**Pull request merged**